test/specs/dom/index.js

```diff
@@ -183,7 +183,6 @@ if (typeof window !== 'undefined') {
 
                   setTimeout(function() {
                     setTimeout(() => { expect(historySpy.callCount).to.equal(1); done() });
-                    done();
```

Comment:
Could you break line 185, so the expect would have its own?

Reply:
I'll format the whole code later after 216 ends.